app/models/like.rb

```diff
@@ -0,0 +1,2 @@
+class Like < ApplicationRecord
```

Comment:
WE are missing some associations here.
Hint: Associations similar to those of comments